vpr/src/place/place_constraints.h

```diff
@@ -88,4 +88,18 @@ inline bool floorplan_legal(const t_pl_blocks_to_be_moved& blocks_affected) {
  */
 void load_cluster_constraints();
 
+/*
+ * Marks blocks as fixed if they have a constraint region that specifies exactly one x, y,
+ * subtile location as legal.
+ * Marking them as fixed indicates that they cannot be moved during initial placement and simulated annealing
+ */
+void mark_fixed_blocks();
+
+/*
+ * Number of tiles covered by a region
```

Comment:
Comment doesn't match what is happening right now, also add more detail about what is passed in and what happens (what is returned)